test-utils/src/main/groovy/net/neoforged/gradle/utils/test/extensions/FileExtensions.groovy

```diff
@@ -0,0 +1,14 @@
+package net.neoforged.gradle.utils.test.extensions
+
+/**
+ * Extensions for the [File] class.
+ */
+class FileExtensions {
```

Comment:
Missing compilestatic

Reply:
Can't be compilestatic, causes compiler errors in consumers.